tests/custom_lookups/tests.py

```diff
@@ -739,3 +739,31 @@ def test_instance_related_lookup(self):
         with register_lookup(article_author, RelatedMoreThan):
             self.assertEqual(article_author.get_lookup("rmt"), RelatedMoreThan)
         self.assertIsNone(article_author.get_lookup("rmt"))
+
+    def test_unregister_class_lookup(self):
+        msg = (
+            "Unsupported lookup 'startswith' for CharField or join on the "
+            "field not permitted, perhaps you meant istartswith?"
+        )
+        author_name = Author._meta.get_field("name")
+        with unregister_lookup(models.CharField, StartsWith):
```

Comment:
You should test it directly against `CharField` as well, not only against instances of `CharField`.

```suggestion
        with unregister_lookup(models.CharField, StartsWith):
            self.assertIsNone(CharField.get_lookup("startswith"))
```

Reply:
`get_lookup()` isn't a class method. Maybe something like `self.assertNotIn("startswith", models.CharField.get_class_lookups())`? Please help me by explaining the purpose of the assertion.